WebApp/Views/Browser/Index.cshtml

```diff
@@ -42,11 +42,21 @@
                 <span class="browser_error_details_info_black">@Strings.BrowserTrust</span>
             </div>
         </div>
+        <div id="errorContextPanel" class="overlay-content">
+            <div  class="browser_error_contextpanel_container">
+                <span class="browser_error_contextpanel_header">@Strings.BrowserErrorMessage</span>
+                <div id="errorHeader" class="browser_error_contextpanel_message"></div>
+                <br />
+                <span class="browser_error_contextpanel_header">@Strings.AlertDetailsDescriptionHeader</span>
```

Comment:
This localized string just contains the word 'description'. I reused an already localized string instead of creating a new one